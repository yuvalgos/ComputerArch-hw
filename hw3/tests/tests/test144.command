./dflow_calc tests/test144.dat tests/test144.in d30 p167 p118 p152 p66 p170 p98 d192 d45 p183 d10 p23 p130 d94 d167 p195 d167 p19 p12 p12 p41 d19 d100 p190 d137 p88 p153 p44 p11 p132 p178 p84 d119 p99 d74 p166 d5 d29 d142 p20 d134 p87 d90 p110 p119 d6 d174 d116 p61 d87 p168 p109 d122 p104 d72 p52 d101 d84 p186 d71 d150 d181 p188 p96 p135 p160 d73 d63 d183 p78