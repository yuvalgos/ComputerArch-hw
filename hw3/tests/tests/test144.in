0 8 16 30
0 18 3 23
3 13 23 18
4 28 18 20
4 29 26 9
3 7 12 8
4 7 5 23
3 2 7 7
3 11 26 26
5 4 2 15
5 28 4 2
6 31 30 11
5 26 26 19
5 20 5 30
4 10 5 18
7 18 17 12
7 19 24 22
4 28 6 15
4 30 20 31
6 7 28 0
7 20 17 31
5 11 23 14
1 2 24 14
6 10 26 31
4 0 20 3
2 0 25 23
7 13 8 18
0 17 10 8
5 18 14 22
0 8 28 25
4 24 18 16
5 9 17 28
6 11 4 29
5 16 20 4
0 11 26 8
6 18 31 15
3 21 24 26
5 22 21 22
1 29 13 10
7 29 16 4
3 26 27 0
1 27 22 11
3 16 4 19
5 5 6 31
2 5 6 23
7 14 9 31
8 8 2 16
3 23 12 4
7 1 30 1
7 6 30 19
2 18 30 9
2 1 21 3
5 11 12 25
5 24 10 28
2 31 27 10
7 18 7 21
3 27 19 24
0 9 28 29
1 18 5 1
1 27 17 9
2 2 6 12
3 3 12 14
4 9 7 13
3 8 12 19
2 14 26 18
6 30 22 9
6 14 26 9
2 11 29 26
5 19 29 1
4 18 6 24
7 19 14 26
3 29 10 21
0 20 8 19
4 10 5 13
1 2 14 8
1 9 12 3
3 28 14 6
7 4 17 22
7 3 11 12
6 31 6 7
5 28 20 0
0 19 2 2
6 21 20 1
1 26 26 22
4 15 5 6
7 1 19 28
0 21 27 2
1 18 25 3
2 19 24 28
0 7 1 30
3 28 17 15
0 3 12 1
7 27 0 28
6 16 12 7
7 7 16 8
0 22 23 9
3 6 10 25
8 26 28 14
0 3 30 25
8 1 11 15
2 4 10 8
6 27 22 8
6 8 16 20
1 28 31 22
7 19 6 11
5 6 18 3
8 16 15 28
8 20 30 21
6 10 1 11
5 4 1 30
0 15 20 18
1 12 15 4
2 4 23 3
1 15 5 2
4 1 23 31
0 11 28 11
8 25 20 26
1 27 11 24
4 13 31 15
5 25 9 10
1 9 5 20
7 15 31 23
4 31 5 19
7 25 25 22
5 12 30 26
4 20 0 1
8 28 30 20
7 6 21 9
4 1 7 29
3 31 11 31
4 10 22 24
6 3 23 23
6 24 27 24
5 4 22 13
0 19 28 27
6 13 7 8
3 29 1 22
2 3 1 15
5 27 23 26
7 17 3 23
6 30 1 21
2 18 7 19
1 19 22 19
6 26 22 10
1 18 13 27
1 22 14 2
7 30 2 5
1 25 7 11
8 9 30 17
2 29 13 24
1 12 13 1
2 19 15 22
5 12 18 24
8 25 3 16
6 9 18 2
7 22 14 0
6 20 9 3
8 31 16 22
6 0 12 2
7 21 25 6
6 15 16 15
6 15 12 10
7 7 1 27
1 30 31 28
0 13 28 30
8 29 28 30
0 27 18 5
5 17 12 14
6 16 11 5
6 27 27 22
3 26 31 18
1 6 5 17
0 9 31 21
7 5 16 10
6 31 18 3
2 14 8 26
3 25 21 16
8 10 27 4
2 30 29 11
0 1 20 0
7 22 26 26
2 2 22 11
3 31 5 18
7 11 17 24
5 20 21 7
7 2 29 3
7 17 15 0
6 5 0 1
6 2 7 5
2 6 19 29
8 15 17 10
3 10 9 9
2 6 15 9
7 17 7 28
5 4 12 26
5 25 29 0
3 12 6 1
6 12 5 29
1 5 16 29
5 8 20 26
