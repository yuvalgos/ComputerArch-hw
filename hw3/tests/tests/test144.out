Reading the opcodes latency info from tests/test144.dat ... Got latency for 9 opcodes
Reading the program file tests/test144.in ... Found 200 instructions
getProgDepth()==261
getInstDeps(30)=={28,-1}
getDepDepth(167)==190
getDepDepth(118)==72
getDepDepth(152)==177
getDepDepth(66)==49
getDepDepth(170)==177
getDepDepth(98)==66
getInstDeps(192)=={190,172}
getInstDeps(45)=={31,11}
getDepDepth(183)==203
getInstDeps(10)=={9,7}
getDepDepth(23)==24
getDepDepth(130)==87
getInstDeps(94)=={93,63}
getInstDeps(167)=={152,66}
getDepDepth(195)==207
getInstDeps(167)=={152,66}
getDepDepth(19)==11
getDepDepth(12)==0
getDepDepth(12)==0
getDepDepth(41)==44
getInstDeps(19)=={17,-1}
getInstDeps(100)=={73,63}
getDepDepth(190)==226
getInstDeps(137)=={128,121}
getDepDepth(88)==100
getDepDepth(153)==173
getDepDepth(44)==0
getDepDepth(11)==7
getDepDepth(132)==95
getDepDepth(178)==162
getDepDepth(84)==54
getInstDeps(119)=={75,108}
getDepDepth(99)==80
getInstDeps(74)=={66,63}
getDepDepth(166)==177
getInstDeps(5)=={-1,0}
getInstDeps(29)=={17,-1}
getInstDeps(142)=={95,134}
getDepDepth(20)==24
getInstDeps(134)=={126,117}
getDepDepth(87)==99
getInstDeps(90)=={27,84}
getDepDepth(110)==105
getDepDepth(119)==120
getInstDeps(6)=={-1,-1}
getInstDeps(174)=={144,137}
getInstDeps(116)=={107,97}
getDepDepth(61)==75
getInstDeps(87)=={-1,78}
getDepDepth(168)==92
getDepDepth(109)==103
getInstDeps(122)=={44,104}
getDepDepth(104)==80
getInstDeps(72)=={63,70}
getDepDepth(52)==0
getInstDeps(101)=={95,63}
getInstDeps(84)=={44,49}
getDepDepth(186)==207
getInstDeps(71)=={23,36}
getInstDeps(150)=={135,128}
getInstDeps(181)=={180,115}
getDepDepth(188)==230
getDepDepth(96)==49
getDepDepth(135)==157
getDepDepth(160)==113
getInstDeps(73)=={44,26}
getInstDeps(63)=={-1,16}
getInstDeps(183)=={167,132}
getDepDepth(78)==80
